### ISSUE TITLE
Can we use import-collection to import a ZIP of Zip's

### ISSUE THREAD

**Issue by craigwalkeruk** [Backbase/bb-cli#91]
When releasing through DTAP we'd like to import items via continuous delivery by using a versioned package/ZIP that will be built once and used across multiple environments.

suggested approach:
- bb build-collection generates all ZIP's and a zip of zips
- extend import-item to import from a ZIP directly with flag
- extend import-collection to unpack a zip of zip's and call the import-item for each with flag

Note: another task needs to extend authentication via HTTPS without clear text passwords

**Comment by sorohan** [comment]
I just noticed this task - I guess it's similar to what I proposed in #97.

I think we just need to extend import-collection to unpack a zip (doesn't have to be of zips) and call import-item for each. I was going to make a separate issue for importing a collection zip after #97 was implemented.

@craigwalkeruk I think this is one of the more important issues we need to focus on so we can setup a proper continuous delivery. Can we assign someone to this?

**Comment by santam85** [comment]
I notice this only now, I think if we properly address #91 and #97 we fix one of the most annoying pain points of import-item. Happy to work on this in the coming weeks...

**Comment by sorohan** [comment]
I have implemented it in a separate library and have been working with it (see @backbase/import). The plan is to integrate that library into bb-cli. I can work with you on that if you like.
